build.gradle

```diff
@@ -7,7 +7,7 @@ plugins {
   id 'org.owasp.dependencycheck' version '6.2.2'
```

Comment:
```suggestion

```
Since we're bringing in a higher [version of dependencyCheck](https://github.com/hmcts/rse-gradle-java-plugin/blob/c6131758379eaf7307f306a2e45ab3702c421642/build.gradle#L13) from uk.gov.hmcts.java, do we need still need this line?